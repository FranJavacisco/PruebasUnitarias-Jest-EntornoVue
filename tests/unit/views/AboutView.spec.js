import { shallowMount } from '@vue/test-utils';
import HomeView from '@/views/AboutView.vue';

describe('AboutView.vue', () => {
    it('matches the snapshot', () => {
        const wrapper = shallowMount(HomeView);
        expect(wrapper.html()).toMatchSnapshot();
    });
});
