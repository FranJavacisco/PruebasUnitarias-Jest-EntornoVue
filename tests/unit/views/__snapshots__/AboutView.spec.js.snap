// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView.vue matches the snapshot 1`] = `
<div class="home"><img alt="Vue logo" src="">
  <posts-view-stub></posts-view-stub>
</div>
`;
